netbox_slm/migrations/0005_add_software_license.py

```diff
@@ -46,6 +46,10 @@ class Migration(migrations.Migration):
                 ('software_product', models.ForeignKey(on_delete=django.db.models.deletion.PROTECT, to='netbox_slm.softwareproduct')),
                 ('tags', taggit.managers.TaggableManager(through='extras.TaggedItem', to='extras.Tag')),
                 ('version', models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.PROTECT, to='netbox_slm.softwareproductversion')),
+                ('license_key', models.CharField(blank=True, max_length=255, null=True)),
```

Comment:
You should generate a new migration (can be done with the `makemigrations` django command) instead of adding to this one. Existing v1.4 installations of the plugin will otherwise not have these fields.

Reply:
I will push it with the next commit